Assets/mParticle/MParticle.cs

```diff
@@ -182,7 +182,20 @@ public enum UserIdentity
 		FacebookCustomAudienceId,
 		Other2,
 		Other3,
-		Other4}
+		Other4,
+		Other5,
+		Other6,
+		Other7,
+		Other8,
+		Other9,
+		Other10,
+		MobileNumber,
+		PhoneNumber2,
+		PhoneNumber3,
+		IOSAdvertiserId,
+		IOSVendorId,
+		PushToken,
+		DeviceApplicationStamp}
```

Comment:
i'm very confused by the formatting here - seems like you're staying consistent to what is in the file, but what's in the file is totally weird - can we fix it?

Reply:
What would an example of your preferred format be? @samdozor